store/actions.js

```diff
@@ -19,7 +19,7 @@ import {
   SET_MSIG_ACTOR_ADDRESS
 } from './actionTypes'
 import getAddressFromReceipt from '../utils/getAddrFromReceipt'
-import { setMessageInCache } from '../utils/cacheMessage'
+// import { setMessageInCache } from '../utils/cacheMessage'
 
```

Comment:
This is only used by one instance of the function, which is also commented out by someone else. Says "temp commented out" so im commenting this as well to make the linter happy.